R/zchunk_LA119.solar.R

```diff
@@ -29,10 +29,37 @@ module_gcam.usa_LA119.Solar_DISABLED <- function(command, ...) {
     # Load required inputs
     states_subregions <- get_data(all_data, "gcam-usa/states_subregions")
     NREL_us_re_capacity_factors <- get_data(all_data, "gcam-usa/NREL_us_re_capacity_factors")
-    NREL_us_re_technical_potential <- get_data(all_data, "gcam-usa/NREL_us_re_technical_potential")
+    NREL_us_re_technical_potential <- get_data(all_data, "gcam-usa/NREL_us_re_technical_potential")   #This data is not being used in this chunk
```

Comment:
Then delete this line as well as the corresponding line above (under `driver.DECLARE_INPUTS`)